src/controllers.ts

```diff
@@ -0,0 +1,62 @@
+import express, { Request, Response } from 'express';
+import * as fs from 'fs';
+
+export class Controller {
+  static path: string = './newFile.txt';
+  static enc: string = 'utf-8';
+
+  static readFile(request: Request, response: Response): void {
+    fs.readFile(Controller.path, Controller.enc, (error, data) => {
+      if (error) {
+        response.status(404).send('File not found');
+        return console.error(error.message);
+      }
+
+      response.status(200).send(data);
+    });
+  }
+
+  static createFile(request: Request, response: Response): void {
+    if (request.body) {
```

Comment:
Error first approach)
Try process negative first of all to avoid multiple if-else constructions.
Example: if (!request.body) return 'FAIL!!!'